src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/ICompilationExtensions.cs

```diff
@@ -211,5 +211,11 @@ public static ImmutableArray<IAssemblySymbol> GetReferencedAssemblySymbols(this
 
         public static INamedTypeSymbol? DisallowNullAttribute(this Compilation compilation)
             => compilation.GetTypeByMetadataName(typeof(DisallowNullAttribute).FullName!);
+
+        public static INamedTypeSymbol? UnmanagedCallingConventionType(this Compilation compilation, string callConv)
+        {
+            var corLibrary = compilation.GetSpecialType(SpecialType.System_Object).ContainingAssembly;
```

Comment:
❔ Is this a requirement?

Reply:
@sharwell At compiler-side, yes.

https://github.com/dotnet/roslyn/blob/eb95c2f45074343b6fa06e723e8dc4cba59d1d6a/src/Compilers/CSharp/Portable/Symbols/FunctionPointers/FunctionPointerMethodSymbol.cs#L210